build.psm1

```diff
@@ -648,11 +648,6 @@ Fix steps:
     if ($PSModuleRestore) {
         Restore-PSModuleToBuild -PublishPath $publishPath
     }
-
-    # Restore the Pester module
-    if ($CI) {
-        Restore-PSPester -Destination (Join-Path $publishPath "Modules")
-    }
```

Comment:
I think this is still useful -- I want to build powershell and then manually run some Pester tests, but not `Start-PSPester`. In this case, I can run `Start-PSBuild -Clean -CI` and Pester will be installed.

Reply:
Maybe I misunderstood the usage of this line. Because I made `Start-PSPester` restore Pester if needed, I thought, I could remove this line. I would be happy to revert this change but would also be interested in why one would want/need Pester to be installed when building PowerShell (my guess is that your use case is to call `Start-DevPowerShell` and run the pester test from there?)

Reply:
> would also be interested in why one would want/need Pester to be installed when building PowerShell (my guess is that your use case is to call Start-DevPowerShell and run the pester test from there?)

When I'm wringing Pester tests for some code changes and just want to run the tests I'm working on, I would build the code base, start `pwsh` from the bin folder, and run `Invoke-Pester -Script <path-of-my-test-file>`. Or when I'm investigating a test failure from daily build, I might want to sync master branch, build, start the `pwsh` produced from the build, run `Invoke-Pester` with the file where the test fails.

Sure, we can install Pester to user module path, so it will always be available, but install the Pester module while build powershell could be useful in case you don't have Pester in other module paths.

Reply:
Ok, I see. I will revert this change then.
When I run tests locally, then my workflow is `Start-PSBuild ...` and then `Start-PSPester $TestFile`

Reply:
Yeah, sometimes I expect some of my tests to fail initially, and it's easier for me to keep the `pwsh` instance around for some ad-hoc verifications.